### ISSUE TITLE
Cookie banner should have earlier focus

### ISSUE THREAD

**Issue by tobiasberge** [shopware/shopware#14787]
### Shopware Version

v6.7.7.0

### Affected area / extension

Platform(Default)

### Actual behaviour

The Cookie Policy does not receive focus until the footer. It should instead receive focus first on the page because it requires immediate action. In the screenreader you also don't navigate to it unless you are in the footer already.

### Expected behaviour

The cookie banner is the first focusable element when no cookie preference has been selected.

### How to reproduce

* Open the Storefront with default theme
* Delete existing Cookies and reload
* The Cookie Banner is only focusable when you navigate/TAB through the entire page first
